app/views/static/deploy-provider-review-progress.html.haml

```diff
@@ -0,0 +1,23 @@
+%div{"ng-controller" => "containers.deployProviderReviewProgressController"}
+  %div{"allow-click-nav" => "false", "miq-wizard-substep" => "", "on-show" => "onShow", "step-id" => "review-progress",
+  "step-priority" => "1", "step-title" => "Progress", "wz-disabled" => "false"}
+    .blank-slate-pf{"ng-if" => "deployInProgress"}
+      .spinner.spinner-lg
+      %h1 Deployment in Progress
+      %p
+        Your deployment
+        %strong {{data.providerName}}
+        is currently in progress. Updates will be posted here shortly.
+      %p If you choose to close this wizard, deployment status can be monitored under requests.
+    .blank-slate-pf{"ng-if" => "deployComplete"}
+      .alert.alert-success{"ng-if" => "deploySuccess"}
+        %span.pficon.pficon-ok
+        Deployment
```

Comment:
This cannot be split because of i18n, need to do `= _("Deployment %{name} was sucessful") % {:name => "{{data.providerName}}""}